test/resolver/strategies/openapi-3-1-apidom/index.js

```diff
@@ -291,6 +291,17 @@ describe('resolve', () => {
           });
         });
       });
+
+      describe('and referenced schema contains allOf keyword', () => {
+        test('should contain schema with merged properties from allOf', async () => {
```

Comment:
Not sure if this test will work properly - it only generated the snapshot after I changed it to `test.only(...)`, so not sure if it tests it without it.